packages/components/organisms/f-cookie-banner/test/specs/component/desktop/f-cookieConsentBanner.component.spec.js

```diff
@@ -34,9 +39,33 @@ describe('New - Multi-tenant - f-cookieBanner component tests', () => {
         // ['no', 'no/informasjonskapselerklaering']  'dk' and 'no' disabled for now
     ])
     .it('should go to the correct cookie policy page', (tenant, expectedCookiePolicyUrl) => {
-
         // Arrange
-        cookieBanner.open(tenant);
+        const countryFormatted = tenant.toUpperCase();
```

Comment:
It'd make this a forEach() test and pass the full tenant code (e.g en-IE / da-DK /  it-IT / nb-NO).

Will tidy the test up a bit 👍🏼